src/main/java/task/NewOpportunityPortal/ability/entity/UserAbility.java

```diff
@@ -0,0 +1,19 @@
+package task.NewOpportunityPortal.ability.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.sql.Timestamp;
+
+@Data
+@AllArgsConstructor
```

Comment:
Please change @AllArgsConstructor and @Data to @Value and remove private modificators